database.js

```diff
@@ -0,0 +1,25 @@
+import { client } from './connections.js';
+
+export const dbName = 'CallOfDuty';
+export const dbCollection = 'Soldiers';
+
+async function addNewSoldier(newSoldier) {
+  newSoldier.duties = [];
```

Comment:
usually not good idea, but in this case it ok

Reply:
fixed